pcm_fitModelCrossval.m

```diff
@@ -179,7 +179,11 @@
     end;
 
     % Now do the fitting
-    fcn = @(x) pcm_groupLikelihood(x,YY,M(m),Z,X,P,'runEffect',B,'S',S);
+    if (M(m).numGparams==0) 
```

Comment:
This addition is to submit G in place of M structure when estimating likelihoods.